utils/build-script

```diff
@@ -934,6 +950,22 @@ the number of parallel build jobs to use""",
     if args.extra_swift_args:
         build_script_impl_args += ["--extra-swift-args", ";".join(args.extra_swift_args)]
 
+    if args.cross_compile_targets:
+        # Currently only cross-compiling the swift libraries is supported.
+        # Building the swift compiler would require building llvm too.
+        llvm_path = os.path.join(build_dir, "llvm-"+args.host_target, 'bin')
+        swift_path = os.path.join(build_dir, "swift-"+args.host_target, 'bin')
+        build_script_impl_args += ["--cross-compile-tools-deployment-targets="
+            + " ".join(args.cross_compile_targets),
+            "--native-llvm-tools-path=" + llvm_path,
+            "--native-clang-tools-path=" + llvm_path,
+            "--native-swift-tools-path=" + swift_path,
+            "--skip-build-cmark=1",
```

Comment:
Does this require a two-step build-script invocation?  We should never require that.  The whole point of build-script is to orchestrate the whole build, taking care of the details.